code_generator/End2End/Tests/NonBlittableComponentTests.cs

```diff
@@ -0,0 +1,88 @@
+using Generated.Improbable.TestSchema.Nonblittable;
+using Improbable.Gdk.Core;
+using NUnit.Framework;
+using Unity.Mathematics;
+using UnityEngine;
+
+namespace Improbable.Gdk.CodeGenerator.End2EndTests
+{
+    [TestFixture]
+    public class NonBlittableComponentTests
+    {
+        private static readonly bool1 Bool1False = new bool1(false);
+        private static readonly bool1 Bool1True = new bool1(true);
+
+        private const int IntValue = 123;
+        private const long LongValue = 5678L;
+        private const float FloatValue = 1.2345f;
+        private const double DoubleValue = 3.14159;
+        private static readonly bool1 BoolValue = Bool1True;
```

Comment:
same question here: do we need `BoolValue`?

Reply:
Answered above.